particles/WasmTemplate/example.cc

```diff
@@ -0,0 +1,71 @@
+#include <arcs.h>
+#include <example-arcs.h>  // generated by schema2packager; filename is based on the manifest file
+
+// C++ wasm builds are currently limited to a single source file per module, but they can
+// define multiple particles.
+
+class BasicParticle : public arcs::Particle {
+public:
+  BasicParticle() {
+    registerHandle("foo", foo_);  // required for each handle declared in the manifest
+    registerHandle("bar", bar_);
+    autoRender();  // automatically render once handles are ready, and then when updated
```

Comment:
Does a particle author have to call this? What if they don't? Maybe it's not needed if a particle is just doing data processing? Adding more to the comment about this could be helpful, or for SDK docs @piotrswigon

Reply:
No, they don't. The arcs header file already has a reasonable level of info on the base Particle methods; I didn't want to replicate all of that here.

However, using autoRender() makes writing many particles much simpler, so it was useful to include in this basic example. (fyi I have a todo to port this feature back to the JS particle.)